src/drivers/device/integrator.h

```diff
@@ -69,7 +69,7 @@ class Integrator
 	 * @param integral_dt	Get the dt in us of the current integration (only if reset).
 	 * @return		the integral since the last read-reset
 	 */
-	math::Vector<3>		get(bool reset, uint64_t &integral_dt);
+	math::Vector<3>		get(bool reset, uint64_t *integral_dt);
```

Comment:
We really need to get every target we care about building in travis-ci or
jenkins. If the eagle toolchain isn't something that we can throw on
travis-ci could you add it to jenkins?

On Thu, Apr 28, 2016 at 5:37 AM, Julian Oes <notifications@github.com>
wrote:

> In src/drivers/device/integrator.h
> <https://github.com/PX4/Firmware/pull/4366#discussion_r61399175>:
>
> > @@ -69,7 +69,7 @@ class Integrator
> >  	 * @param integral_dt	Get the dt in us of the current integration (only if reset).
> >  	 * @return		the integral since the last read-reset
> >  	 */
> > -	math::Vector<3>		get(bool reset, uint64_t &integral_dt);
> > +	math::Vector<3>		get(bool reset, uint64_t *integral_dt);
>
> May I ask why this change? Any particular reason?
>
> It broke make eagle_default.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/4366/files/ec4d2fa8de963e1d9379aa32e57b1723619b7b0d#r61399175>
>